_pages/history.md

```diff
@@ -0,0 +1,104 @@
+[SafariWatir](https://github.com/redsquirrel/safariwatir), and
+[ChromeWatir](https://github.com/jijeshmohan/chromewatir)).
+
+Selenium also had the advantage of easily supporting development in multiple programming languages. 
+As a result, Watir was also adapted into Java with [Watij](https://sourceforge.net/projects/watij/) and
+.NET with [Watin](https://www.codeproject.com/Tips/658947/Watin-An-Automation-Testing-in-NET).
+
+WebDriver was created by Simon Stewart in 2007 (also at ThoughtWorks at that time) and it provided
+solutions to some of the major issues Selenium faced. Rather than applying a single 
+JavaScript implementation for all browsers, it required creating a unique driver for each browser
+that would have more power and flexibility. The code base boasted a cleaner API and was less of a testing
+framework than a toolkit of direct browser automation 
+commands.(<strong>[6](http://www.seleniumhq.org/about/history.jsp)</strong>) 
+
+As the WebDriver drivers matured, Selenium found the WebDriver solutions appealing enough to agree to 
+merge the projects.
+While Selenium 2.0 effectively emulated Selenium 1.0 (RC) for backward compatibility, adopting WebDriver’s 
+object oriented driver approach required significant code updates.
+Since Watir had always used the "driver" approach, Watir users ironically had much fewer adjustments to make in
+their code than Selenium users did in order to be able to leverage the full power of WebDriver.
+
+When Jari Bakken, a Watir developer, was exposed to WebDriver in 2009, he began developing a Ruby implementation of it 
+for both Selenium and Watir. Jari had already written [Celerity](https://github.com/jarib/celerity), 
+a JRuby project to implement a wrapper for HtmlUnit, to provide a headless testing option with the same API as Watir. 
+With his work on watir-webdriver, he decided to take the Celerity specs and make them usable for both 
+projects.([7](https://github.com/watir/watirspec/tree/9443ea77ab998b4f676a120221b460dcab421933)). 
+In doing so he created the WatirSpec repository which could be used by any of the Ruby-based projects that 
+implemented the Watir API.
+
+The functionality being provided by WebDriver caused the Watir team to add features and change its conventions 
+when it made sense to do so. The big backward-incompatible change in the [Watir 2.0](http://watir.com/watir-2-0/) 
+(released in August of 2011) was the move from one-indexing to zero-indexing. 
+Indexing “like a person” makes sense until you have to constantly figure out if you are working with a 
+Watir Collection or a Ruby Array (or with WebDriver). The challenge of bringing Watir into complete 
+compliance with the WatirSpec was taken up by Jarmo Pertman and culminated in the release 
+of [Watir 3.0](http://watir.com/watir-3-released/) in April 2012.
+
+To encourage the use of watir-webdriver, Hugh McGowan renamed the Watir code base “watir-classic” and 
```

Comment:
@titusfortner This all was also done by me. Maybe Hugh had some part in it, but I can't remember exactly. Where did you get this information about him being associated with Watir 4?

Reply:
Hmm, I've heard him say several times that his greatest contribution to the project was getting the watir code renamed to watir-classic...
https://github.com/watir/watir-classic/commit/de8068013b20a5bdc762b4802b02ab2bce1b65ea
, and it's his commits at the beginning of the meta gem. Looks like that was back in March & April of 2012, but that Watir 4 wasn't released until September, which I'm guessing was you. Can't exactly tell what was going on between April & September

Reply:
Ok, it's obvious looking at the code that you did all the difficult work there. :)
Reworded the content here.

Reply:
Oh, looking at the code, you are correct that he did that gem, but I took over. There was also something about commonwatir - can't remember why was that gem used for. Maybe it was a code, which worked with Watir and Firewatir at the time? And maybe Hugh did that gem to glue these gems together somehow? Looking here https://github.com/watir/watir_meta/commit/7f6306ecf2193c8870c390fc8eb08769095e61f1 I've starting to remember - watir meta-gem was also used for releasing new versions of gems so that if watir-classic version got bumped, then commonwatir version got bumped as well and they were released at the same time. It basically started paving the way from migrating from watir-classic to watir-webdriver in the end and simplified release process of all the different, but related gems.

Reply:
Here's an explanation for commonwatir - https://github.com/watir/watir_meta/commit/c92689e9134c35e2a57dcd517f0b6a6d60fd5d3a